FDK/Tools/SharedData/FDKScripts/CheckOutlinesUFO.py

```diff
@@ -12,1020 +13,1145 @@
-if fixed.
+-setMinArea <n>
+    Set the minimum area for a tiny outline. Default is 25 square
+    units. Subpaths with a bounding box less than this will be reported, and
+    deleted if fixed.
 
--setTolerance <n>  Set the maximum deviation from a straight line allowed.
-Default is 0 design space units. This is used to test whether the control points
-for a curve define a flat curve, and whether a sequence of line segments
-defines a straight line.
+-setTolerance <n>
+    Set the maximum deviation from a straight line allowed.
+    Default is 0 design space units. This is used to test whether the control
+    points for a curve define a flat curve, and whether a sequence of line
+    segments defines a straight line.
 
--wd	write changed glyphs to default layer instead of '%s'
+-wd
+    write changed glyphs to default layer instead of '%s'
 
 -g <glyphID1>,<glyphID2>,...,<glyphIDn>
-	Check only the specified list of glyphs. The list must be
-	comma-delimited. The glyph ID's may be glyphID's, glyph names, or
-	glyph CID's. If the latter, the CID value must be prefixed with the
-	string "cid". There must be no white-space in the glyph list.
-	Examples:
-		checkOutlinesUFO -g A,B,C,69 myFont
-		checkOutlinesUFO -g cid1030,cid34,cid3455,69 myCIDFont
+    Check only the specified list of glyphs. The list must be
+    comma-delimited. The glyph ID's may be glyphID's, glyph names, or
+    glyph CID's. If the latter, the CID value must be prefixed with the
+    string "cid". There must be no white-space in the glyph list.
+    Examples:
+        checkOutlinesUFO -g A,B,C,69 myFont
+        checkOutlinesUFO -g cid1030,cid34,cid3455,69 myCIDFont
 
 -gf <file name>
-	Check only the list of glyphs contained in the specified file, The file
-	must contain a comma-delimited list of glyph identifiers. Any number of
-	space, tab, and new-line characters are permitted between glyph names
-	and commas.
+    Check only the list of glyphs contained in the specified file, The file
+    must contain a comma-delimited list of glyph identifiers. Any number of
+    space, tab, and new-line characters are permitted between glyph names
+    and commas.
 
--all Force all glyphs to be processed. This applies only to UFO fonts, where processed glyphs
-	are saved in a layer, and processing of a glyph is skipped if has already been processed.
+-all
+    Force all glyphs to be processed. This applies only to UFO fonts, where
+    processed glyphs are saved in a layer, and processing of a glyph is skipped
+    if has already been processed.
 
 """ % kProcessedGlyphsLayerName
 
 __doc__ = __usage__ + __help__
 
-
 import sys
 import os
 import re
-import copy
 import defcon
 import ufoLib
 import subprocess
 import shutil
 import ufoTools
 import booleanOperations.booleanGlyph
 from robofab.pens.digestPen import DigestPointPen
-from plistlib import readPlist, writePlistToString
 
 import hashlib
 
+nameAliasDict = {}
+
 kSrcGLIFHashMap = "com.adobe.type.checkOutlinesHashMap"
```

Comment:
this isn't used, it seems

Reply:
It's referenced in expand_names(), but I think that code may not be getting exercised, because it was indicating nameAliasDict as global, but there was no declaration of nameAliasDict in the previous version, and the usage there seems to be write only -- I don't see it getting read anywhere. Shall I go ahead and remove all references to nameAliasDict?

Reply:
I meant the `kSrcGLIFHashMap` variable.
As for `nameAliasDict` I+d change it to `name_alias_dict` for now.

Reply:
OK, yes, I agree that kSrcGLIFHashMap does not appear to be used. I will remove it.

As for nameAliasDict/name_alias_dict -- why not remove it?  Seems to be write-only.

Reply:
Remove it then

Reply:
Will do.